src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -2351,6 +2351,7 @@ internal enum ErrorCode
 
         ERR_IteratorRefLikeElementType = 9267,
         ERR_CollectionExpressionKeyValuePairNotSupported = 9268,
+        ERR_WithElementMustBeFirst = 9269,
```

Comment:
Would it make sense to remove the `With` and name this more generally? Perhaps `ERR_CollectionArgumentsMustBeFirst`.

Reply:
Your error codes have no rhyme or reason to them :).  In my case, this was about a syntactic issue reported during binding, so i wanted it to be specific.